blueocean-dashboard/src/main/js/redux/actions.js

```diff
@@ -227,6 +327,26 @@ exports.fetchLogsInjectStart = function fetchLogsInjectStart(url, start, onSucce
             }
         });
 };
+
+/**
+ * Determines if the provided object is a "run" type
+ */
+function isRun(obj) {
+    return obj._class === 'io.jenkins.blueocean.rest.impl.pipeline.PipelineRunImpl'
```

Comment:
cc @vivek - is there a list that can be looked at, or is it best to either look at the code or hit the api and see what it returns and guess from the names what the capabilities return look like?